build/release-notes.md

```diff
@@ -7,6 +7,90 @@ toc_max: 2
 
 This page contains information about the new features, improvements, and bug
 fixes in [Docker Buildx](https://github.com/docker/buildx){:target="blank" rel="noopener" class=""}.
+## 0.11.0
+
+{% include release-date.html date="2023-06-13" %}
+
+### New
+
+- Bake now supports [matrix builds](../build/bake/reference.md#targetmatrix).
+  The new matrix field on `target` lets you create multiple similar targets to
+  remove duplication in bake files. [docker/buildx#1690](https://github.com/docker/buildx/pull/1690)
+- New experimental `--detach` flag for running builds in detached mode.
+  [docker/buildx#1296](https://github.com/docker/buildx/pull/1296),
+  [docker/buildx#1620](https://github.com/docker/buildx/pull/1620),
+  [docker/buildx#1614](https://github.com/docker/buildx/pull/1614),
+  [docker/buildx#1737](https://github.com/docker/buildx/pull/1737),
+  [docker/buildx#1755](https://github.com/docker/buildx/pull/1755)
+- New experimental [debug monitor mode](https://github.com/docker/buildx/blob/v0.11.0-rc1/docs/guides/debugging.md)
+  that lets you start a debug session in your builds.
+  [docker/buildx#1626](https://github.com/docker/buildx/pull/1626),
+  [docker/buildx#1640](https://github.com/docker/buildx/pull/1640)
+- New [`EXPERIMENTAL_BUILDKIT_SOURCE_POLICY` environment variable](./building/env-vars.md#buildkit_experimental_source_policy)
+  for applying a BuildKit source policy file.
+  [docker/buildx#1628](https://github.com/docker/buildx/pull/1628)
+
+### Bug fixes and enhancements
```

Comment:
So looking at the previous release notes on this page, it looks like we've summarized only the main things in those ones?

I think we can remove all the `panics`, `hangs`, `leaks`, etc. We can always link to the full release notes for more info for the users who care to click through.

Reply:
pruned some of the fixes, and added a link to the GH release note. PTAL